### PR TITLE
Better erroring

### PR DESCRIPTION
**Pull request by gmarceau**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,14 +4,29 @@ var util = require('util'),
...
+                _recur(v);
+            }
+        });
+    }
+    _recur(config);
 };
-config.util.substituteEnv(config);
+
+substituteEnv(config);
 config.util.makeImmutable(config);
```
Comment (sashaverma) [inline index.js:30]: do we use substituteEnv anywhere outside of this file? just double-checking dependencies.

**Comment by sashaverma** [comment]
👍

File: index.js (hunk 1 above)
Comment (gmarceau) [inline index.js:30]: No, it's private.

Comment (sashaverma) [inline index.js:30]: Coolies, I just remembered a few days back when we were calling that so just wanted to double-check. :)

**Pull request merged**